react/MinicartContext.tsx

```diff
@@ -75,7 +80,11 @@ const MinicartContextProvider: FC<Props> = ({
     hasBeenOpened: false,
     openOnHoverProp,
     openBehavior:
-      resolvedVariation === 'popup' && openOnHoverProp ? 'hover' : 'click',
+      (resolvedVariation === 'popup' ||
+        resolvedVariation === 'popupWithLink') &&
```

Comment:
```suggestion
      checkVariationIsPopup(resolvedVariation) &&
```

Reply:
Hello Igor,

Code adjusted, could you check, please?